wagtail/search/backends/database/sqlite/query.py

```diff
@@ -272,9 +269,7 @@ def normalize(search_query: SearchQuery) -> Tuple[SearchQuery]:
         ):  # If there are no negated subqueries, return an Or().
             return Or(normalized_subqueries)
 
-        for (
-            subquery
-        ) in (
+        for subquery in (
             negated_subqueries
         ):  # If there's a MatchAll subquery, then anything will get matched.
```

Comment:
```suggestion
        for subquery in negated_subqueries:
            # If there's a MatchAll subquery, then anything will get matched.
```

Reply:
Ah, good catch. It looks like `ruff format` is not formatting these lines due to the trailing comment. I'll move the comments to a newline and it should format these into single line statements.